rest/taskrouter/workflows/list/post/example-1/example-1.5.x.php

```diff
@@ -0,0 +1,56 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require_once '/path/to/vendor/autoload.php'; // Loads the library
+
+use Twilio\Rest\Client;
+
+// Your Account Sid and Auth Token from twilio.com/user/account
+$accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+$authToken = "your_auth_token";
+$workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+$client = new Client($accountSid, $authToken);
+
+$salesQueue = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+$marketingQueue = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+$supportQueue = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+$everyoneQueue = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+$defaultTarget = new WorkflowRuleTarget($everyoneQueue);
```

Comment:
That's great. Is there a plan to add this to the next-gen version of the library?